go.mod

```diff
@@ -1,20 +1,22 @@
 module github.com/Asherda/lightwalletd
 
-go 1.12
+go 1.13
 
 require (
-	github.com/Asherda/Go-VerusHash v0.0.0-20200820044955-b08dfc22193f
+	github.com/DavidLDawes/Go-VerusHash v0.0.0-20200825015559-eaf28ab6d352
```

Comment:
Side effect of building & testing using my archives. 2 further steps needed after merging this PR, one right away, one once we get to VerusCoin/lightwalletd:

1. Once this PR merges, create another PR for ASherda's master branch replacing DavidLDawes & the details with the Asherda/Go-Verushash details; this also requires at least the first Asherda/Go-VerusHash release.
2. Once we go to VerusCoin/lightwalletd we will need to PR a replacement yet again this time with the VerusCoin/Go-Verushash values. Again this also requires at least the first VerusCoin/Go-VerusHash release.